portfolio/src/main/webapp/hobbies.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>My Hobbies</title>
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
+    </head>
+    <body>
+      <div id="barTop">
+        <nav class="navbar navbar-expand-lg navbar-dark" style="background-color: #004047">
+          <a class="navbar-brand" href="#">Andrés's Portfolio</a>
+          <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+            <div class="navbar-nav">
+            <a class="nav-item nav-link active" href="index.html">Home <span class="sr-only">(current)</span></a>
+            <a class="nav-item nav-link active" href="#">Projects</a>
```

Comment:
Are you planning on making another page just for projects? Right now this option is pointing to this hobbies page. If you would like it to point to a Project page in the future, I would leave the href blank, and then leave a TODO. This makes it easier for the reviewer to understand and also reminds future Andrés to update it :)

https://google.github.io/styleguide/htmlcssguide.html#Action_Items

Reply:
Thanks for the TODO heads up!